### ISSUE TITLE
Bug: Uncaught TypeError: Cannot read property 'style' of undefined when adding new tab

### ISSUE THREAD

**Issue by yawalkar** [pojome/elementor#3889]
## Description

When trying to add a new tab with tabs widget, I get this error in console - 

`Uncaught TypeError: Cannot read property 'style' of undefined - editor.js?ver=4.9.4:100`

## Steps to reproduce

Add a new Tabs widget and click on "Add Item" button. Here's the screencast - 

![image](https://d2ffutrenqvap3.cloudfront.net/items/2E3o2m0v1a2T1W3B2A2S/Screen%20Recording%202018-03-26%20at%2012.01%20AM.gif)

## Environment

```
					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.13.5
	MySQL version: 5.5.5
	PHP Version: 7.1.10-1+ubuntu14.04.1+deb.sury.org+1
	PHP Max Input Vars: 10000
	PHP Max Post Size: 1024M
	GD Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.9.4
	Site URL: http://dev.local
	Home URL: http://dev.local
	WP Multisite: No
	Max Upload Size: 1 GB
	Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Active

== Theme ==
	Name: Astra
	Version: 1.2.6
	Author: Brainstorm Force
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.162 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 1.9.8
		Author: Elementor.com

	Jetpack by WordPress.com
		Version: 5.9
		Author: Automattic

	Query Monitor
		Version: 2.17.0
		Author: John Blackbourn

	Regenerate Thumbnails
		Version: 3.0.2
		Author: Alex Mills (Viper007Bond)

	WooCommerce
		Version: 3.3.4
		Author: Automattic

	WooCommerce PayPal Express Checkout Gateway
		Version: 1.5.2
		Author: WooCommerce

	WooCommerce Tab Manager
		Version: 1.8.4
		Author: SkyVerge

	WordPress Reset
		Version: 1.4
		Author: Matt Martz


== Debug ==
	Errors: 
		TypeError: Cannot read property 'style' of undefined
			at http://dev.local/wp-admin/js/editor.js?ver=4.9.4 - 100:39
			2018-03-25 18:27 +00:00
			x 7 times

		TypeError: name.toLowerCase is not a function
			at http://dev.local/wp-includes/js/jquery/jquery-migrate.js?ver=1.4.1 - 102:23
			2018-03-25 18:21 +00:00

		TypeError: Cannot read property 'style' of undefined
			at http://dev.local/wp-admin/js/editor.js?ver=4.9.4 - 100:39
			2018-03-25 18:14 +00:00
			x 3 times

		TypeError: name.toLowerCase is not a function
			at http://dev.local/wp-includes/js/jquery/jquery-migrate.js?ver=1.4.1 - 102:23
			2018-03-25 18:13 +00:00

		TypeError: Cannot read property 'style' of undefined
			at http://dev.local/wp-admin/js/editor.js?ver=4.9.4 - 100:39
			2018-03-25 18:12 +00:00

		TypeError: name.toLowerCase is not a function
			at http://dev.local/wp-includes/js/jquery/jquery-migrate.js?ver=1.4.1 - 102:23
			2018-03-25 18:12 +00:00

		TypeError: Cannot read property 'style' of undefined
			at http://dev.local/wp-admin/js/editor.js?ver=4.9.4 - 100:39
			2018-03-25 18:12 +00:00

		TypeError: name.toLowerCase is not a function
			at http://dev.local/wp-includes/js/jquery/jquery-migrate.js?ver=1.4.1 - 102:23
			2018-03-25 18:12 +00:00

		TypeError: Cannot read property 'style' of undefined
			at http://dev.local/wp-admin/js/editor.js?ver=4.9.4 - 100:39
			2018-03-25 18:12 +00:00

		TypeError: name.toLowerCase is not a function
			at http://dev.local/wp-includes/js/jquery/jquery-migrate.js?ver=1.4.1 - 102:23
			2018-03-25 18:12 +00:00

```

**Comment by bainternet** [comment]
you have JS/PHP/PLUGIN/THEME conflict somewhere in your install.

Start by deactivating all other add-ons and plugins other than Elementor Core & Elementor Pro (if installed) - then proceed to edit and see if the issue has been resolved. You may need to switch to the default theme to rule out your theme as the cause of the conflict.

If the issue is resolved by following the above action the proceed to reactivate each plugin 1 at a time and checking the editor till you find the one causing the conflict.

if not, Please contact our support at support@elementor.com for guidance in resolving this issue.

**Comment by yawalkar** [comment]
Well, that didn't helped. I already have tried deactivating all other plugins and with twentytwelve default theme. Also tried with the latest version released today 2.0, but the error is still there.

Sent the email to the email id mentioned above. Any kind of help would be greatly appreciated.

Thank you,
Nitin

**Comment by yawalkar** [comment]
I figured out the issue, and it was the Slider control in repeater. Opened a new issue here - https://github.com/pojome/elementor/issues/3919

Hope you'll resolve it soon 👍
